releasenotes/notes/remove-ddtrace-utils-ae7231d2e4f130b2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    The deprecated module ``ddtrace.utils`` is removed.
```

Comment:
Same here, there are a bunch of submodules that might be worth calling out how to upgrade